docs/sdks/kotlin-sdk/integrate-with-okta.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Integrate With Okta
+sidebar_position: 2
+---
+
+This guide describes how to configure Beyond Identity using Okta as the OIDC authentication flow.
+
+### Prerequisites
+
+ - [Integrate With Okta](../../sso-integrations/integrate-with-okta)
+ - [Kotlin SDK](overview)
+
+Before calling [`EmbeddedSdk.authenticate()`](overview#authentication), we must [Authorize With Okta](#authorize-with-okta)
+
+### Authorize With Okta
+
+ - Step 1: Okta URL
+
+To start the authorization flow, launch a [`WebView`](https://developer.android.com/reference/android/webkit/WebView), and load the Oauth2 Authorization Request URL provided by Okta.
+
+![Okta Identity Provider](../screenshots/Okta%20Identity%20Provider.png)
+
+```
+val webView = WebView(activity)
+...
+webView.loadUrl(OKTA_URL)
+```
+
+ - Step 2: Invoke URL
+
+Create a [`WebViewClient`](https://developer.android.com/reference/android/webkit/WebViewClient) and override [`shouldOverrideUrlLoading`](https://developer.android.com/reference/android/webkit/WebViewClient#shouldOverrideUrlLoading(android.webkit.WebView,%20android.webkit.WebResourceRequest)). A URL with the Invoke URL scheme should be returned from Okta. When the webpage loads the URL, call [`EmbeddedSdk.authenticate()`](overview#authentication). You can confirm the validity of the URL with [`EmbeddedSdk.isAuthenticateUrl()`](overview#authenticate-url-validation).
+
+```
```

Comment:
Let's add javascript for our code snippets, just to get some sort of syntax highlighting for now.